proposal-selection.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: default
+---
+
+# Proposal selection process
```

Comment:
The more I read this, the more I'm convinced that this should be changed to 

# Google Season of Docs proposal selection process

There is a lot of reference to GSoD in this document that is fine if the document isn't trying to masquerade as our own internal project selection process.

Reply:
I'm happy to let this ride in favour of releasing what is basically an alpha version of the guidelines for us to rapidly iterate on.